playbook-website/app/javascript/site_styles/_kits_show.scss

```diff
@@ -17,4 +18,11 @@
       align-self: center;
     }
   }
+  .kearning {
+    letter-spacing: $lspace_looser;
+  }
+  .card {
+    max-width: fit-content;
+    display: inline-block;
+  }
```

Comment:
Anything we can do to use global props and eliminate the need for CSS?

Reply:
@nidaqg is trying a few things out now on this one.